test/core/iomgr/endpoint_pair_test.cc

```diff
@@ -44,7 +124,16 @@ static grpc_endpoint_test_fixture create_fixture_endpoint_pair(
   a[0].type = GRPC_ARG_INTEGER;
   a[0].value.integer = static_cast<int>(slice_size);
   grpc_channel_args args = {GPR_ARRAY_SIZE(a), a};
+#ifdef GRPC_POSIX_SOCKET_TCP
```

Comment:
Same comment regarding non-posix-specific functionality.

Reply:
Removed now.

Reply:
The experiment checking is now done inside src/core/lib/event_engine/shim{.h,.cc} files. The ifdef statements are only added there.